x-pack/plugins/security_solution/public/common/components/and_or_badge/rounded_badge_antenna.tsx

```diff
@@ -40,10 +39,6 @@ const BottomAntenna = styled(EuiFlexItem)`
   }
 `;
 
-const EuiFlexItemWrapper = styled(EuiFlexItem)`
-  margin: 0 12px 0 0;
```

Comment:
Removed, ended up being unnecessary. This antenna version of the badge is only being used in exceptions UI, so would not have ramifications elsewhere.